packages/ERTP/src/issuer.chainmail

```diff
@@ -0,0 +1,209 @@
+/**
+ * The issuer cannot mint new amount, but it can create empty purses and
+ * payments. The issuer can also transform payments (splitting payments,
+ * combining payments, burning payments, and claiming payments
+ * exclusively). The issuer should be gotten from a trusted source and
+ * then relied upon as the decider of whether an untrusted payment is valid. 
+ */
+interface Issuer (Amount (Extent)) {
+  /**
+   * Get the Brand for this Issuer. The Brand indicates the kind of
+   * digital asset and is shared by the mint, the issuer, and any purses
+   * and payments of this particular kind. The brand is not closely
+   * held, so this function should not be trusted to identify an issuer
+   * alone. Fake digital assets and amount can use another issuer's brand.
+   */
+  getBrand() -> (Brand);
```

Comment:
This file should also declare the `Brand` api.

Reply:
Good idea!